python/pyhie/allspark/model/llama_v4.py

```
@@ -504,6 +508,14 @@ def _build_graph(self, torch_cfg, derive_type):
         if self.do_dynamic_quantize_convert:
             for op in graph.ops:
                 quantize_op(op, self.quant_config, self.quantize_map)
+        # FP8 blockwise: convert GEMM ops to GemmFP8Blockwise for native FP8
+        if getattr(self, 'is_fp8_blockwise', False) and not self.do_dynamic_quantize_convert:
+            for op in graph.ops:
+                if (op.op_type.upper().startswith("GEMM")
+                        and op.weights
+                        and op.weights[0].name in self.weight_name_map):
+                    quantize_gemm_fp8_blockwise(op, self.fp8_block_size)
```

Flake8 (7.3.0)

[error] 'quantize_gemm_fp8_blockwise' may be undefined, or defined from star imports: ..quantization, .model_base, .quantization_utils

(F405)